executor/tests/services/test_workspace_service.py

```
@@ -0,0 +1,236 @@
+        assert should_exclude("src/main.py") is False
+        assert should_exclude("package.json") is False
+        assert should_exclude("README.md") is False
+
+
+class TestGetGitTrackedFiles:
+    """Test get_git_tracked_files function."""
+
+    def test_get_files_success(self, tmp_path):
+        """Test getting git-tracked files from a git repo."""
+        # Create a git repo with some files
+        subprocess.run(["git", "init"], cwd=tmp_path, capture_output=True)
```

Ruff (0.14.14)

[error] Starting a process with a partial executable path

(S607)